src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -126,6 +148,28 @@ public void createUser(final DBUser user, final HashedPassword hashedPassword) {
   }
 
   @Override
+  public void createUser(final DBUser user, final String password) {
+    Preconditions.checkState(user.isValid(), user.getValidationErrorMessage());
+
+    try (final Connection con = connectionSupplier.get()) {
+      try (final PreparedStatement ps = con.prepareStatement(
+          "insert into ta_users (username, password, email, joined, lastLogin, admin) values (?, ?, ?, ?, ?, ?)")) {
+        ps.setString(1, user.getName());
+        ps.setString(2, BCrypt.hashpw(password, BCrypt.gensalt()));
+        ps.setString(3, user.getEmail());
+        ps.setTimestamp(4, new Timestamp(System.currentTimeMillis()));
+        ps.setTimestamp(5, new Timestamp(System.currentTimeMillis()));
+        ps.setInt(6, user.isAdmin() ? 1 : 0);
+        ps.execute();
+      }
+    } catch (final SQLException e) {
```

Comment:
The connection needs to be committed on the preceding line.  You probably want to review all the new SQL code and make sure the connection is committed in each case.